src/components/HeaderAuth.jsx

```diff
@@ -50,10 +45,9 @@ HeaderAuth.defaultProps = {
   initialised: false,
 };
 
-function mapStateToProps(state) {  // Listens for changes in the Redux Store
-  return {
-    user: state.login.user,
-    initialised: state.login.initialised,
-  };
-}
+const mapStateToProps = (state) => ({
```

Comment:
Out of interest, why does this assign an anonymous closure to a variable, rather than declaring `mapStateToProps` as a function?